jupyterhub/values.schema.yaml

```diff
@@ -2757,6 +2757,12 @@ properties:
           See [the Kubernetes
           documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
           for more details about annotations.
+      customPaths:
+        type: array
+        description: |
+          A list of custom paths to be added to the ingress configuration. 
+          Each custom path allows traffic to be routed to a specific backend service.
+          For each path, you can specify the service name, and either a service port number or a service port name.
```

Comment:
Since the admin has to essentially provide the raw K8s config for a path can you add a link to the relevant part of the K8s docs?

Reply:
sure i'll add the URL to the kubernetes doc